pypesto/store/read_from_hdf5.py

```diff
@@ -120,10 +121,9 @@ def read(self, objective: ObjectiveBase = None) -> Problem:
         if objective is None:
             objective = Objective()
             # raise warning that objective is not loaded.
-            logger.info(
-                'WARNING: You are loading a problem.\nThis problem'
-                ' is not to be used without a separately created'
-                ' objective.'
+            warnings.warn(
```

Comment:
why not logger.warn? 🤔

Reply:
Because then I can't easily silence just this warning